ml-proto/test/br_if.wast

```diff
@@ -14,10 +14,10 @@
     (block (call $dummy) (call $dummy) (br_if 0 (get_local 0)))
   )
   (func (export "as-block-first-value") (param i32) (result i32)
-    (block i32 (br_if 0 (i32.const 10) (get_local 0)) (return (i32.const 11)))
+    (block i32 (drop (br_if 0 (i32.const 10) (get_local 0))) (return (i32.const 11)))
```

Comment:
The `return` is also in that block, i.e. with indentation it looks like this:

````
(block $block0 i32
  (br_if $block0
    (i32.const 10)
    (get_local $0)
  )
  (return
    (i32.const 11)
  )
)
````